### PR TITLE
migrate to indy grails plugin

### PR DESCRIPTION
**Pull request by kananindzya**
refer #1337

### CODE REVIEW

File: apm-agent-plugins/apm-grails-plugin/src/main/java/co/elastic/apm/agent/grails/GrailsTransactionNameInstrumentation.java [hunk 1]
```diff
@@ -114,7 +107,6 @@ static void setTransactionName(@Advice.Argument(2) Object handler) {
             setName(transaction, className, methodName);
         }
 
-        @VisibleForAdvice
         public static void setName(Transaction transaction, String className, @Nullable String methodName) {
```
Comment (felixbarny) [inline apm-agent-plugins/apm-grails-plugin/src/main/java/co/elastic/apm/agent/grails/GrailsTransactionNameInstrumentation.java:110]:
```suggestion
        private static void setName(Transaction transaction, String className, @Nullable String methodName) {
```

**Review by felixbarny — APPROVED** [review]

File: apm-agent-plugins/apm-grails-plugin/src/main/java/co/elastic/apm/agent/grails/GrailsTransactionNameInstrumentation.java (hunk 1 above)
Comment (kananindzya) [reply apm-agent-plugins/apm-grails-plugin/src/main/java/co/elastic/apm/agent/grails/GrailsTransactionNameInstrumentation.java:110]: with private - this method not visible:
```
.UrlMappingsInfoHandlerAdapter
java.lang.IllegalStateException: private static void co.elastic.apm.agent.grails.GrailsTransactionNameInstrumentation$HandlerAdapterAdvice.setTransactionName(java.lang.Object) is not visible to class org.grails.web.mapping.mvc.UrlMappingsInfoHandlerAdapter
	at net.bytebuddy.asm.Advice$Dispatcher$Delegating$Resolved.bind(Advice.java:8887) ~[byte-buddy-dep-1.10.12.jar:?]
```

Comment (felixbarny) [reply apm-agent-plugins/apm-grails-plugin/src/main/java/co/elastic/apm/agent/grails/GrailsTransactionNameInstrumentation.java:110]: `setTransactionName` has to remain public but I think `setName` can be private.

Comment (kananindzya) [reply apm-agent-plugins/apm-grails-plugin/src/main/java/co/elastic/apm/agent/grails/GrailsTransactionNameInstrumentation.java]: done

**Review by SylvainJuge — APPROVED** [review]

**Pull request merged**